dimasacat_andronicus/dimasacat_andronicus.html

```diff
@@ -0,0 +1,247 @@
+          <p>
+            "Andro thank you for helping me on some coding session and 
+            lesson. Always remember you have us THE VRZN team always 
+            got you."
+          </p>
+          <p>- Adrian Naoe</p>
+        </div>
+      </div>
+    </div>
+    <div class="container-gallery">
+      <h2>Gallery</h2>
+      <div class="guide-gallery">
+        <img
+          class="image-gallery"
+          src="./images/dain.png"
+          alt="Dainsleif Image from Genshin Impact"
+        />
+        <img
+          class="image-gallery"
+          src="./images/mclarenp1.jpg"
+          alt="A Mclaren P1 Sports Car Image "
+        />
+        <img
+          class="image-gallery"
+          src="./images/paris.jpg"
+          alt="Eiffel Tower Image"
+        />
+        <img
+          class="image-gallery"
+          src="./images/landscape.jpg"
+          alt="Beautiful Landscape Image"
+        />
+        <img
+          class="image-gallery"
+          src="./images/Lady.jpg"
+          alt="Lady in the Night Street Image"
+        />
+        <img
+          class="image-gallery"
+          src="./images/dialga.jpg"
+          alt="Dialga Image from Pokemon Anime"
+        />
+      </div>
+    </div>
+  </body>
+  <section class="container-goals">
+    <h2>My Goals</h2>
+    <div class="guide-goals">
+      <div class="goals-style">
+        <p>Obtain a Professional Certification</p>
+        <p>
+          IT certifications, like CompTIA A+, Cisco CCNA, or AWS Certified
+          Solutions Architect, can enhance one's career prospects and
+          knowledge.
+        </p>
+      </div>
+      <div class="goals-style">
+        <p>Increase Technical Proficiency</p>
+        <p>
+          Enhancing technical skills and expertise in areas such as cloud
+          computing, cybersecurity, or data analytics is a common goal.
+        </p>
+      </div>
+      <div class="goals-style">
+        <p>Stay Current with Technology Trends</p>
+        <p>
+          Staying updated with the latest industry trends, such as 
+          artificial intelligence, blockchain, or IoT, is essential 
+          to remain competitive in the IT field.
+        </p>
+      </div>
+      <div class="goals-style">
+        <p>Take Masteral for Artificial Intelligence</p>
+        <p>
+          This educational journey offers a unique opportunity to 
+          contribute to cutting-edge research, gain hands-on 
+          experience, and position myself for impactful roles.
+        </p>
+      </div>
+    </div>
+  </section>
+  <div class="container-comment">
+    <div class="guide-comment">
+      <h2>Comment Section</h2>
+      <button 
+        id="sort_button" 
+        onclick="sortOrder()">
+        Sort Order
+      </button>
+    </div>
+    <div class="comment-input-style">
+      <input
+        id="input_name"
+        type="text"
+        placeholder="Input your name"
+        oninput="validate()"
+      />
```

Comment:
refer to this [Guideline](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27)

Reply:
I don't know why it happened like that. I already made changes to all your review. Thanks.